rviz_ogre_vendor/rviz_ogre_vendor-extras.cmake.in

```diff
@@ -48,7 +48,7 @@ foreach(_lib IN LISTS OGRE_LIBRARIES)
       NO_CMAKE_SYSTEM_PATH
       NO_CMAKE_FIND_ROOT_PATH
     )
-    if(NOT _ogre_main_static_library_debug_abs AND APPLE)
+    if(NOT _ogre_main_static_library_debug_abs AND NOT WIN32)
```

Comment:
Is it not append on Linux too?

Reply:
No, this changed with the newest release.

Reply:
Ok, can you guys rebase this and rerun the CI? I'll have another look tomorrow.

Reply:
Will do, thanks for the review!